ui/components/app/signature-request-siwe/signature-request-siwe-message/index.scss

```diff
@@ -0,0 +1,13 @@
+.signature-request-siwe-message {
+  flex: 1 100%;
+  border-radius: 8px;
+  padding: 8px 16px;
+  margin: 16px;
+  border: 1px solid var(--color-border-muted);
+
+  .sub-text {
```

Comment:
can we update this to `.signature-request-siwe-message_sub-text`? The reason is to avoid name collision. Best to stay away from generic names unless we're writing utility classes or we're working with them through third-party libraries

Reply:
Updated